kubernetes/apps/charts/postgresql-backup/values.yaml

```diff
@@ -7,3 +7,6 @@ secretMounts: []
 config:
   envFrom: []
   env: []
+
+healthchecks:
+  host: https://healthchecks.ops.k8s.jarv.us
```

Comment:
Should there be a `calitp.org` host configured for this?

Reply:
I think not in this case; we're adding healthchecks as an option for deployed apps, but we'd deploy a new Cal-ITP specific instance and point the apps at that calitp.org URL in the event of a handoff.